### ISSUE TITLE
interactive task lists work incorrectly

### ISSUE THREAD

**Issue by Haruka-Kono** [executablebooks/jupyterlab-myst#199]
## Description
Hi, when I clicked checkboxes of rendered task lists, wrong responses happened (e.g., could not uncheck and checked no-clicking boxes) although using classical writing method for the lists was OK, as an attached gif animation.

My env related to jupyter:
```
OS: opensuse-leap 15.4 on wsl2 (win11)
Browser: brave or edge
Python: 3.10.13

Selected Jupyter core packages...
IPython          : 8.20.0
ipykernel        : 6.29.0
ipywidgets       : 8.1.1
jupyter_client   : 8.6.0
jupyter_core     : 5.7.1
jupyter_server   : 2.12.5
jupyterlab       : 4.0.11
nbclient         : 0.8.0
nbconvert        : 7.14.2
nbformat         : 5.9.2
notebook         : 7.0.7
qtconsole        : 5.5.1
traitlets        : 5.14.1
```
How can I solve it?

![tasklists](https://github.com/executablebooks/jupyterlab-myst/assets/34228051/5f2f759f-b44b-4bb0-b748-54411a864b82)

**Comment by agoose77** [comment]
Thanks for reporting this! I'll add it to my task list :) ;)

**Comment by nthiery** [comment]
Thanks for reporting this as well; we just stumbled on the same issue this week here;
this way I don't have to report it :-)

**Comment by agoose77** [comment]
Fixed by #202
